argoverse/evaluation/stereo/utils.py

```diff
@@ -0,0 +1,160 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+"""Stereo utilities to support the Argoverse stereo evaluation."""
+
+from pathlib import Path
+from typing import List
+
+import cv2
+import disparity_interpolation
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+from argoverse.evaluation.stereo.constants import (
+    DEFAULT_ABS_ERROR_THRESHOLDS,
+    DEFAULT_REL_ERROR_THRESHOLDS,
+    LOG_COLORMAP,
+)
+
+
+def compute_disparity_error(
+    pred_fpath: Path,
+    gt_fpath: Path,
+    gt_obj_fpath: Path,
+    figs_fpath: Path,
+    abs_error_thresholds: List[int] = DEFAULT_ABS_ERROR_THRESHOLDS,
+    rel_error_thresholds: List[float] = DEFAULT_REL_ERROR_THRESHOLDS,
+    save_disparity_error_image: bool = False,
+) -> pd.DataFrame:
+    """Compute the disparity error metrics."""
+    pred_disparity = cv2.imread(str(pred_fpath), cv2.IMREAD_ANYCOLOR | cv2.IMREAD_ANYDEPTH)
+    gt_disparity = cv2.imread(str(gt_fpath), cv2.IMREAD_ANYCOLOR | cv2.IMREAD_ANYDEPTH)
+    gt_obj_disparity = cv2.imread(str(gt_obj_fpath), cv2.IMREAD_ANYCOLOR | cv2.IMREAD_ANYDEPTH)
+
+    pred_disparity = np.float32(pred_disparity) / 256.0
+    gt_disparity = np.float32(gt_disparity) / 256.0
```

Comment:
I might consider making a constant for `256.0`: `RGB_NORMALIZATION: int = 2 ** 8`

Reply:
Correction: `RGB_NORMALIZATION: float = 2.0 ** 8`

Reply:
Well, whatever you think is best. I guess `int` makes sense as long as it doesn't interfere with any casting.

Reply:
Yeap, changed to `DISPARITY_NORMALIZATION: float = 2.0 ** 8`